lib/ui/platform_dispatcher.dart

```diff
@@ -1333,7 +1340,13 @@ class ViewConfiguration {
   /// relative to.
   ///
   /// If null, then this configuration represents a top level view itself.
-  final FlutterView? window;
+  @Deprecated('''
+      Use the `view` property instead.
+  ''')
+  FlutterView? get window => _view;
+  FlutterView? get view => _view;
```

Comment:
`view` needs a doc comment.

Reply:
I think that this is a duplicate of https://github.com/flutter/engine/pull/38453#discussion_r1060960086